Ginger/Ginger/RunSetPageLib/NewRunSetPage.xaml.cs

```diff
@@ -468,8 +468,8 @@ private void UpdateReRunFailedButtonIcon(bool isRunStarted = false)
                     }
                     else
                     {
-                        xReRunFailedRunsetBtn.ButtonText = "ReRunFailed";
-                        xReRunFailedRunsetBtn.ToolTip = "Run All Failed Runners";
+                        xReRunFailedRunsetBtn.ButtonText = "ReRunFailedFlows";
```

Comment:
Change button text to "Re-Run Failed Flows"

Reply:
done